UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -865,8 +920,8 @@ public class Tile : UndertaleObject, RoomObject, INotifyPropertyChanged
             public UndertaleSprite SpriteDefinition { get => _spriteDefinition.Resource; set { _spriteDefinition.Resource = value; OnPropertyChanged(); OnPropertyChanged("ObjectDefinition"); } }
 
             /// <summary>
-            /// From which object this tile stems from.
-            /// Will return a <see cref="UndertaleBackground"/> if <see cref="spriteMode"/> is disabled, a <see cref="UndertaleSprite"/> if it's enabled.
+            /// From which object this tile stems from.<br/>
+            /// Will return a <see cref="UndertaleBackground"/> if <see cref="spriteMode"/> is true, a <see cref="UndertaleSprite"/> if it's false.
```

Comment:
Use langwords for true and false please.

Reply:
Yeah, I thought about it.